sql/mysql/inspect.go

```diff
@@ -911,6 +911,12 @@ type (
 		Values []string
 	}
 
+	// NetworkType stores an IPv4 or IPv6 address.
+	NetworkType struct {
+		schema.Type
+		T string
+	}
```

Comment:
Please keep only parsing/types tests and remove the integrations ones. No need to run all our tests on another database just for this new type.